service/generators.gradle

```diff
@@ -34,6 +34,14 @@ idea.module.generatedSourceDirs = [file(swaggerOutputSrc)]
 sourceSets.main.java.srcDir swaggerOutputSrc
 compileJava.dependsOn generateSwaggerCode
 
+// Copy the result of the openapi build into the built resources
+// so it is available for the swagger_ui.html endpoint
+task copyOpenApiToResources(type: Copy) {
+    description 'Copy the openapi results into build resources to support swagger_ui'
+    from '../common/openapi.yml'
```

Comment:
It looks like we've now repeated `../common/openapi.yml` in three different places. It would be great if it could be replaced with a shared variable. I think defining it in `bio.terra.catalog.java-common-conventions.gradle` would work, the same way we do for `version` and `group`. Also using a relative path here is brittle, if something like `"${buildDir}/common/openapi.yml"` works that would be better.

Reply:
We can also commit a symlink for each of these locations, that seems to be what `external-credentials-manager` does: https://github.com/DataBiosphere/terra-external-credentials-manager/search?q=openapi